tutorials/scripting/scene_unique_nodes.rst

```diff
@@ -40,3 +40,36 @@ name in the path for ``get_node()``. For example:
  .. code-tab:: csharp
 
     GetNode<Button>("%RedButton").Text = "Hello";
+
+Same-scene limitation
+---------------------
+
+A scene unique node can only be retrieved by a node inside the same scene.
+
+If the same scene is instantiated twice, a node inside the first instance
+calling ``get_node("%RedButton")`` will get the button in the first instance,
+not the button in the second instance.
+
+This limitation is perfect for a UI scene like a health bar, where each instance
+of the health bar must not interact with other instances.
+
+When a scene is instantiated inside another scene, the two scenes can't access
+scene unique nodes from each other. If a Player scene contains an instance of a
```

Comment:
Note from a discussion in Discord where scene unique nodes came up again: you can access the scene unique nodes, but you have to navigate into the scene first. Like `get_node("Hand/Sword/%Hilt")`, or `s = get_node("Hand/Sword")` `s.get_node("%Hilt")`. Seems worth noting, to me.